packages/object-deep-omit/index.tests.ts

```diff
@@ -0,0 +1,32 @@
+import deepOmit from './index';
+
+const arr = ['one', 'two'];
```

Comment:
Add case for shallow omit string which should still work 
const obj3 = {foo: 1, bar: 2}
// OK
deepOmit(obj3, 'foo')